src/components/AdminSettings/Federations.vue

```diff
@@ -0,0 +1,80 @@
+<!--
+ - @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ -
+ - @author Maksim Sukharev <antreesy.web@gmail.com>
+ -
+ - @license AGPL-3.0-or-later
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+	<section id="general_settings" class="federations section">
+		<h2>
+			{{ t('spreed', 'Federations') }}
```

Comment:
The thing is called "Federation" (singular there  is no plural of it in English)